sql-blueprinting/README.md

```diff
@@ -0,0 +1,26 @@
+# SQL Blueprinting
+
+Compare two databases and drop the tables that are not common in both.
+We run our strapi project in a new sql db so it creates its clean structure.
+This db can be used as a blueprint since it is created by our strapi current state and doesnt have old entries etc.
+
+## Usage example
```

Comment:
I believe I understand that this should be used with two databases of the same version (eg v3), can you add some clarification to that point as it may confuse some users.

Reply:
Will update the README